openc3/lib/openc3/models/python_package_model.rb

```diff
@@ -92,7 +92,7 @@ def self.install(name_or_path, scope:)
         end
       end
       Logger.info "Installing python package: #{name_or_path}"
-      result = OpenC3::ProcessManager.instance.spawn(["/openc3/bin/pipinstall", "--no-warn-script-location", "-i", pypi_url, package_file_path], "package_install", package_filename, Time.now + 3600.0, scope: scope)
+      result = OpenC3::ProcessManager.instance.spawn(["/openc3/bin/pipinstall", "--no-warn-script-location", "-i", pypi_url, "--trusted-host", URI.parse(pypi_url).host, package_file_path], "package_install", package_filename, Time.now + 3600.0, scope: scope)
```

Comment:
I like having the option to set trusted-host but it shouldn't always be on. That defeats SSL even when you have a good SSL connection.  Maybe add an ENV to enable trusted host?

Reply:
I added an environment variable called `PIP_ENABLE_TRUSTED_HOST` and only apply `--trusted-host` if that environment variable is set.